subprojects/openssl.wrap

```diff
@@ -1,8 +1,8 @@
 [wrap-file]
-directory = openssl-3.0.2
-source_url = https://www.openssl.org/source/openssl-3.0.2.tar.gz
-source_filename = openssl-3.0.2.tar.gz
-source_hash = 98e91ccead4d4756ae3c9cde5e09191a8e586d9f4d50838e7ec09d6411dfdb63
+directory = openssl-3.0.7
+source_url = https://www.openssl.org/source/openssl-3.0.7.tar.gz
+source_filename = openssl-3.0.72.tar.gz
```

Comment:
Hm... I made a typo here, but it still works fine. I'm a bit confused :thinking:

Reply:
It will simply use a weird cache filename in `subprojects/packagecache/`, but the file's remote location still matches `source_url = ` and the contents of the file still match `directory = `.

So it all works.